docs/canvas/canvas-tutorial.asciidoc

```diff
@@ -26,10 +26,14 @@ To customize your workpad to look the way you want, add your own images.
 +
 The default Elastic logo image appears on the page.
 
-. To add your own image, click the Elastic logo, then drag the image file to the *Select or drag and drop an image* field.
+. Add your own image.
+
+.. Click the Elastic logo.
+
+.. Drag the image file to the *Select or drag and drop an image* field.
```

Comment:
Is "field" the correct word here?

Reply:
Yup! It's a field in the configuration panel. This is how we refer to it in Lens too.